shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java

```diff
@@ -231,7 +231,7 @@ public String getCipherColumn(final String logicTable, final String logicColumn)
     public List<Object> getEncryptAssistedQueryValues(final String schemaName, final String logicTable, final String logicColumn, final List<Object> originalValues) {
         Optional<EncryptAlgorithm> encryptor = findEncryptor(schemaName, logicTable, logicColumn);
         Preconditions.checkArgument(encryptor.isPresent() && encryptor.get() instanceof QueryAssistedEncryptAlgorithm,
-                String.format("Can not find QueryAssistedEncryptAlgorithm by %s.%s.", logicTable, logicColumn));
+                String.format("Cannot find QueryAssistedEncryptAlgorithm by %s.%s.", logicTable, logicColumn));
```

Comment:
@cheese8 Can we keep the original format?

Reply:
just to test ci is ready or not